packages/card/style.module.css

```diff
@@ -92,6 +92,20 @@
   }
 }
 
+.logoThumbnail {
+  composes: thumbnail;
+  padding: var(--wpl-spacing-05);
+
+  & .image {
+    aspect-ratio: none;
+
+    & img {
+      aspect-ratio: 16 / 9;
+      object-fit: contain;
+    }
+  }
+}
```

Comment:
Essentially a superset of the thumbnail with logo-specific overrides:
- Padding to prevent logos from touching the thumbnail space's sides
- Moving of the `aspect-ratio` rule from the `.image` wrapper to the `<img>` element doing the rendering
- Changing `<img>`'s `object-fit` rule to `contain` to prevent oversizing